sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
```

Comment:
Why no package_version.hpp change for AMQP?

Reply:
It is at the right version already.